app/src/main/java/com/example/wrk/TrackerActivity.java

```diff
@@ -160,6 +162,7 @@ private void updateUserStreak() {
         if (lastWorkout != null) {
             LocalDate recentWorkout = lastWorkout.toInstant().atZone(ZoneId.systemDefault()).toLocalDate();
             long recentWorkoutTime = recentWorkout.atStartOfDay(ZoneId.systemDefault()).toEpochSecond();
+            lastWorkoutMonth = recentWorkout.getMonthValue();
```

Comment:
might be more readable if you move here 
if (lastWorkoutMonth == today.getMonthValue() {
            workoutsThisMonth++;
        }